wbia/other/ibsfuncs.py

```diff
@@ -3312,7 +3312,7 @@ def get_primary_species_viewpoint(species, plus=0):
         >>> # ENABLE_DOCTEST
         >>> from wbia.other.ibsfuncs import *  # NOQA
         >>> import wbia
-        >>> species = wbia.const.TEST_SPECIES.ZEB_PLAIN
+        >>> species = const.TEST_SPECIES.ZEB_PLAIN
```

Comment:
Missing import for `const`

Reply:
covered by `from wbia.other.ibsfuncs import * which imports const. Can add redundant import for readability though

Reply:
added the import anyway